nilearn/glm/tests/test_contrasts.py

```diff
@@ -20,12 +20,12 @@
     [
         (
             "face / 10 + (window - face) * 2 - house",
-            "a face xy_z house window".split(),
+            ["a", "face", "xy_z", "house", "window"],
```

Comment:
am kinda "meh" on this one but it only happens in single test so sure

Reply:
We really want a list of strings here, why call a function to construct it instead of writing the list as a literal? Sure, the result might be less visually appealing, but that's on Python.

Reply:
yeah I agree but it just looks weird: I will live with it. np